gax-httpjson/src/main/java/com/google/api/gax/httpjson/EmptyMessage.java

```diff
@@ -58,4 +59,62 @@ public Object getFieldValue(String fieldName) {
   public ApiMessage getApiMessageRequestBody() {
     return null;
   }
+
+  @Override
+  public boolean equals(Object other) {
+    return other instanceof EmptyMessage;
+  }
+
+  private static final EmptyMessage DEFAULT_INSTANCE;
```

Comment:
Why not just `private static final EmptyMessage DEFAULT_INSTANCE = new EmptyMessage();`?

Reply:
This just follows the protobuf java paradigm: https://github.com/googleapis/google-cloud-java/blob/c011130c0114ccdcde9bda60202f334722df331d/google-api-grpc/proto-google-cloud-dataproc-v1/src/main/java/com/google/cloud/dataproc/v1/Cluster.java#L2595